modules/rhpam-apb/roles/deploy-kieserver/templates/kieserver-dc.yml.j2

```diff
@@ -305,7 +305,7 @@ spec:
               readOnly: true
 {% if kieserver_db_type == 'H2' %}
             - name: h2-storage
-              mountPath: "/opt/eap/standalone/data"
+              mountPath: "/opt/kie/data"
```

Comment:
If this mountPath is going to be "/opt/kie/data" instead of "/opt/kie/data/h2", then perhaps shouldn't "h2-pvol" get renamed to "kie-pvol", and "h2-claim" get renamed to "kie-claim"?

Reply:
@ruromero @ricardozanini : This PR got merged before I was able to review. We might now need a follow-up PR per my comment above. This would go along with the same comment I made in the templates and operator.

Reply:
I approved and merged the PR because it was technically correct. The mixed mode (addressed by this PR and JIRA) is enabled with the `AUTO_SERVER=TRUE` flag.
The change of the mount path it is not mentioned in the JIRA but doesn't affect the application execution either and that is why I didn't care much about it.
If you prefer to rename the PersistenceVolumeClaim and PersistenceVolume names it can perfectly be done in a follow-up PR as it is not related either with the JIRA attached.

Reply:
@errantepiphany @ruromero  this will be handled in https://issues.jboss.org/browse/KIECLOUD-217. Don't you rather wait for it?

Reply:
That's will work.